static-instrumenter/agent-instrumenter/src/integrationTest/java/JarTest.java

```diff
@@ -43,17 +46,25 @@ void testSampleJar() throws Exception {
             outPath.toString());
 
     Process instrumentationProcess = instrumentationProcessBuilder.start();
-    InputStream instrumentationIn = instrumentationProcess.getErrorStream();
     instrumentationProcess.waitFor(10, TimeUnit.SECONDS);
 
-    String instrumentationLog =
-        new String(instrumentationIn.readAllBytes(), StandardCharsets.UTF_8);
+    boolean isHotJdkJvm = System.getProperty("java.vm.name").contains("OpenJDK");
+    if (isHotJdkJvm) {
+      InputStream errorOutput = instrumentationProcess.getErrorStream();
```

Comment:
Can you use try-with-resources for IO streams?

Reply:
The `close()` of `InputStream` does nothing (see https://docs.oracle.com/javase/8/docs/api/java/io/InputStream.html#close--).

Reply:
Subclasses of InputStream may though